### ISSUE TITLE
reactive statement will never change

### ISSUE THREAD

**Issue by DetachHead** [sveltejs/eslint-plugin-svelte#382]
### Motivation

```svelte
<script>
	let _value = ''
	$: value = getValue()
	
	const getValue = () => _value
</script>

<input bind:value={_value}>
<p>value: {value}</p>
```
here, the `$:` is being used incorrectly as it will not be re-run when `_value` changes, because there are no variables that will change being referenced

when learning svelte, i ran into issues like this several times, so i think there should be a compiler error/warning when reactive statements are used incorrectly like this

### Description

error when all of the variables referenced in a reactive statement are immutable

### Examples

```svelte

<!-- ✓ GOOD -->
<script>
	let _value = ''
	$: value = getValue(_value)
	
	const getValue = (arg) => arg
</script>

<!-- ✗ BAD -->
<script>
	let _value = ''
	$: value = getValue()
	
	const getValue = () => _value
</script>
```

### Additional comments

also raised https://github.com/sveltejs/svelte/issues/8309 but it was closed

**Comment by ota-meshi** [comment]
Thank you for suggesting the new rule. 
That rule sounds good to me.
